src/core/Akka.Remote/MessageSerializer.cs

```diff
@@ -25,12 +25,10 @@ public static class MessageSerializer
         /// <returns>System.Object.</returns>
         public static object Deserialize(ActorSystem system, SerializedMessage messageProtocol)
         {
-            Type type = messageProtocol.HasMessageManifest
-                ? Type.GetType(messageProtocol.MessageManifest.ToStringUtf8())
-                : null;
-            var message = system.Serialization.Deserialize(messageProtocol.Message.ToByteArray(),
-                messageProtocol.SerializerId, type);
-            return message;
+            return system.Serialization.Deserialize(
```

Comment:
What's the impact of this change? Why was it necessary? Not saying it's wrong or criticizing it, I just want to understand what it's for is all.